templates/zerver/integrations/email.md

```diff
@@ -1,8 +1,5 @@
-You can send emails to Zulip! This is useful:
+# Zulip Email integration
 
-  * If you have an email that you want to discuss on Zulip
-  * For mirroring mailing list traffic
-  * For integrating a third-party app that can send emails, but which does not
-    easily lend itself to more direct integration
+You can send emails to Zulip!
 
-Instructions are available [here](/help/message-a-channel-by-email).
+See our help center documentation for [messaging a channel by email](/help/message-a-channel-by-email).
```

Comment:
I have not used tabs or made it an ordered list, as it's just a single line, and I think it looks better this way.